pkg/router/crl/crl.go

```diff
@@ -66,6 +67,9 @@ var (
 	CRLFilename = filepath.Join(mtlsLatestSymlink, crlBasename)
 	// CABundleFilename is the fully qualified path to the currently in use CA bundle.
 	CABundleFilename = filepath.Join(mtlsLatestSymlink, caBundleBasename)
+	// crlsUpdated is true when all CRLs have been successfully updated, and false when there are missing CRLs.
+	crlsUpdated = false
+	crlsMutex   = sync.Mutex{}
```

Comment:
It's better to be very explicit about mutexes.  
```suggestion
	// crlsUpdated is true when all CRLs have been successfully updated, and false when there are missing CRLs.
	// You must take crlsMutex before using crlsUpdated.  
	crlsUpdated = false
	// crlsMutex protects crlsUpdated.  
	crlsMutex = sync.Mutex{}
```